static/js/tools/map/chart_loader.tsx

```diff
@@ -28,6 +28,7 @@ import React, {
   useState,
 } from "react";
 
+import { NO_FULL_COVERAGE_PLACE_TYPES } from "../../constants/map_constants";
```

Comment:
I don't think this import is needed anymore?

Reply:
Thanks for the catch! Removed.